neofetch

```diff
@@ -1228,6 +1228,10 @@ getsong () {
             ;;
         esac
 
+    elif ([ -n "$(ps aux | awk '!(/awk/ || /Helper/) && /Google Play Music Desktop Player/')" ] && [ -n "$(which gpmdp)" ]); then
```

Comment:
Are the `()` brackets needed around the start and end of the elif?

Also, change the `[ -n "$(which gpmdp)" ]` part to:

```sh
type -p gpmdp >/dev/null 2>&1
```